common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/WebViewAuthorizationFragment.java

```diff
@@ -126,6 +127,14 @@ void extractState(@NonNull final Bundle state) {
         mAuthorizationRequestUrl = state.getString(REQUEST_URL);
         mRedirectUri = state.getString(REDIRECT_URI);
         mRequestHeaders = getRequestHeaders(state);
+        // In cases of WebView as an auth agent, we want to always add the passkey protocol header.
+        // (Not going to add passkey protocol header until full feature is ready.)
+        if (FidoConstants.IS_PASSKEY_SUPPORT_READY) {
```

Comment:
put in getRequestHeaders?

Reply:
Done here: 5bc0a09b6ff9d082411ada6f9b99d9f50e249ecc